### PR TITLE
Validating special coverage queries

### PR DESCRIPTION
**Pull request by benghaziboy**

### CODE REVIEW

**Comment by benghaziboy** [comment]
@akos this should prevent that clickhole issue from happening again.

File: tests/special_coverage/test_spec_cov_camp.py [hunk 1]
```diff
@@ -100,3 +100,31 @@ def test_query_string_resolves_to_dict_when_saving(self):
         )
 
         self.assertIsInstance(SpecialCoverage.objects.all()[0].query, dict)
+
+    def test_query_string_filters_null_values_when_saving(self):
+        """Andrew Kos comments on tests. I however, do not."""
```
Comment (benghaziboy) [inline tests/special_coverage/test_spec_cov_camp.py]: @kand

Comment (kand) [inline tests/special_coverage/test_spec_cov_camp.py]: @benghaziboy are you going to turn that into a test or what? Your clarification is clearly not executable.

Comment (benghaziboy) [inline tests/special_coverage/test_spec_cov_camp.py]: I think it is important that I express myself in ways other than code.

Comment (benghaziboy) [inline tests/special_coverage/test_spec_cov_camp.py]: So does my hateful therapist.

**Pull request merged**